src/dynarec/la64/la64_emitter.h

```diff
@@ -1916,7 +1919,9 @@ LSX instruction starts with V, LASX instruction starts with XV.
 #define VPICKVE2GR_DU(rd, vj, imm1)  EMIT(type_2RI1(0b011100101111001111110, imm1, vj, rd))
 #define VFRINT_S(vd, vj)             EMIT(type_2R(0b0111001010011101001101, vj, vd))
 #define VFRINT_D(vd, vj)             EMIT(type_2R(0b0111001010011101001110, vj, vd))
-#define VFRINTRRD_D(vd, vj, imm4)    EMIT(type_2RI4(0b011100101001110101, imm4, vj, vd))
+//  vfrintrm.d 0010 vfrintrm.s 0001, vfrintrne.d 1110 vfrintrne.s 1101...
```

Comment:
As you can see from the encoding, the rounding mode is determined by only 2 bits, not 4. So what I meant was to change `round_round` to

```
    static const int8_t round_round[] = {
        0x3, // round to nearest with ties to even
        0x0, // round toward minus infinity
        0x1, // round toward plus infinity
        0x2  // round toward zero
    };
```

And have the macros be

```
#define VFRINTRRD_S(vd, vj, rm)      EMIT(type_2RI4(0b011100101001110101, ((rm & 0b11) << 2) | 0b01, vj, vd))
#define VFRINTRRD_D(vd, vj, rm)      EMIT(type_2RI4(0b011100101001110101, ((rm & 0b11) << 2) | 0b10, vj, vd))
```

Reply:
Got it, I'll fix it.